Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[workspace]
+resolver = "2"
+
+members = [
+    "forged-rs",
+    "forged-cli",
+]
+
+# Config for 'cargo dist'
+[workspace.metadata.dist]
+# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.9.0"
+# CI backends to support
+ci = ["github"]
+# The installers to generate for each app
+installers = ["shell", "powershell", "homebrew", "msi"]
+# Target platforms to build apps for (Rust target-triple syntax)
+targets = ["aarch64-apple-darwin", "x86_64-apple-darwin", "x86_64-unknown-linux-gnu", "x86_64-pc-windows-msvc"]
+# Publish jobs to run in CI
+pr-run-mode = "plan"
+
+# The profile that 'cargo dist' will build with
+[profile.dist]
+inherits = "release"
+lto = "thin"
+
+[workspace.metadata.dist.dependencies.apt]
```

Comment:
https://github.com/probe-rs/probe-rs/blob/master/Cargo.toml#L69

This is what we do in probe-rs 🤔